pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -484,7 +485,12 @@ private static void setupTaskReceiver(Config cfg, String projectId) {
         if (enableHKeepTasks) {
             var subName = ProjectSubscriptionName.of(projectId, cfg.getString(ConfigFile.PUBSUB_HKEEP_TASKS_SUB));
             var receiver = new HousekeepingTaskReceiver(subName, scheduler);
-            taskSubscriber = Subscriber.newBuilder(subName, receiver).build();
+            taskSubscriber = Subscriber.newBuilder(subName, receiver)
+                    .setParallelPullCount(1)
```

Comment:
Looking at docs it appears that this already defaults to 1. Guess better to be explicit.